sklearn/utils/estimator_checks.py

```diff
@@ -56,8 +56,9 @@
 
 BOSTON = None
 CROSS_DECOMPOSITION = ['PLSCanonical', 'PLSRegression', 'CCA', 'PLSSVD']
-MULTI_OUTPUT = ['CCA', 'DecisionTreeRegressor', 'ElasticNet',
-                'ExtraTreeRegressor', 'ExtraTreesRegressor', 'GaussianProcess',
+MULTI_OUTPUT = ['CCA',  'DecisionTreeClassifier', 'DecisionTreeRegressor',
```

Comment:
![image](https://cloud.githubusercontent.com/assets/15546524/23048339/262f0f24-f4db-11e6-8e44-82a6ffb099f2.png)

These are the errors when I do not include them.